docs/guide/Microfrontend-JAMStack.md

```diff
@@ -1,243 +1,326 @@
+
+### Continuous Integration / Continuous Delivery (CI/CD)
+
+Each micro-frontend has its own CI/CD set up; there is no requirement to share the same setup or use the same CI tool.
+
+The CI/CD can be configured to support multiple environments, for example, DEV, QA, PROD.
+
+Here is an example file showing a git action workflow.
+
+```yml
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [16.x]
+
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+    # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+    - uses: actions/checkout@v2
+
+    # Runs a single command using the runners shell
+    - name: Use NodeJS ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: Cache node modules
+      uses: actions/cache@v2
+      env:
+        cache-name: cache-node-modules
+      with:
+        # npm cache files are stored in `~/.npm` on Linux/macOS
+        path: '**/node_modules'
+        key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/yarn.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-build-${{ env.cache-name }}-
+          ${{ runner.os }}-build-
+          ${{ runner.os }}-
+    - run: yarn install --frozen-lockfile
+    - run: yarn lint
+    - run: yarn test
+    - run: yarn build
+#    - name: Slack Notification
+#      uses: rtCamp/action-slack-notify@v2.0.2
+#      env:
+#        SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
+```
+
+### CDNs
+
+The resulting SPA is served by a CDN or multiple CDNs. Individual micro-frontends can live in different CDNs.
+
+### Kubernetes
+
+The resulting SPA can run and be served in one or more Kubernetes environments.
+
+### Host application
+
+The host application comes with a pre-configured setup out-of-the-box. It does not need any particular configuration different from a traditional SPA more than the Webpack 5 Module Federation configuration.
+It will be acting as the orchestrator, loading the remote micro frontends and providing them with app-wide functionality e.g. auth, RBAC, client side routing.
+
+There is virtually no limit on how the host can grow and how much can be extended.
+It is suggested however to centralize all the host-child communication and shared components in an external library so that both host and children have the same knowledge and integration won’t break.
```

Comment:
```suggestion
It is suggested, however, to centralize all the host-child communication and shared components in an external library so that both host and children have the same knowledge and integration will not break.
```